src/base/net/smtp.cpp

```diff
@@ -89,6 +88,17 @@ namespace
 
         return hostname.toLocal8Bit();
     }
+
+    bool canEncodeAsLatin1(const QStringView string)
+    {
+        for (int i = 0; i < string.length(); ++i)
```

Comment:
maybe use range based for loop? or even `std::all_of`

Reply:
Done.